docusaurus/docs/reference/ha/migrating.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_label: Migrating
+sidebar_position: 30
+---
+
+# Migrating Controllers
```

Comment:
I'd move this down in the list to the bottom. Sure there are some who will need this, but the doc shouldn't focus on them imo